lib/pbkit/pbkit.c

```diff
@@ -79,6 +79,8 @@ struct s_CtxDma
     DWORD               isGr;
 };
 
+static unsigned int pb_ColorFmt = NV097_SET_SURFACE_FORMAT_COLOR_LE_A8R8G8B8;
+static unsigned int pb_DepthFmt = NV097_SET_SURFACE_FORMAT_ZETA_Z24S8;
```

Comment:
Should this be `uint32_t` or something? These are pushbuffer method data fields (which are 32 bit unsigned integers)